server/game/core/Player.js

```diff
@@ -657,20 +657,26 @@ class Player extends GameObject {
      * @param {PlayType} playingType
      * @param card DrawCard
      * @param target BaseCard
+     * @param penaltyAspect Aspect that is not present on the current base or leader
      */
     runAdjustersForCostType(playingType, baseCost, card, target, ignoreType = false, penaltyAspect = null) {
```

Comment:
This method is now mixing cost increase / decrease with aspect penalty negation which is making the logic a little unnecessarily complex. Since there's no situation where those ever overlap right now, probably we should have separate methods with separate logic for regular vs aspect penalty adjustment